e2e/test/scenarios/native/native-reproductions.cy.spec.ts

```diff
@@ -169,8 +169,13 @@ describe("issue 49454", () => {
     });
   });
 
-  it("should be possible to use metrics in native queries (metabase#49454)", () => {
+  it("should be possible to use metrics in native queries (metabase#49454, metabase#51035)", () => {
     H.startNewNativeQuestion();
+
+    cy.log("should not show empty tooltip (metabase#51035)");
+    cy.button("Save").realHover();
+    H.tooltip().should("not.exist");
```

Comment:
I was not able to reproduce the issue in unit tests.
I really didn't want to introduce a new e2e test just for this, so I just squeezed it into an existing one.